### PR TITLE
stats: show same info on one-line as multi-line

### PR DESCRIPTION
**Pull request by howeyc**
#### What is the purpose of this change?

The one-line stats output is not showing the same information as the multiline stats. The change makes the stats show/not-show using the same if-statements as the multiline stats.

#### Was the change discussed in an issue or in the forum before?

No, but it seemed odd to me that doing a delete with --one-line-stats didn't show any information at all. I assumed I would see the same as multiline, but compact. This does that.

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [x] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: fs/accounting/stats.go [hunk 1]
```diff
@@ -374,7 +385,7 @@ func (s *StatsInfo) String() string {
 			_, _ = fmt.Fprintf(buf, "Transferred:   %10d / %d, %s\n",
 				s.transfers, ts.totalTransfers, percent(s.transfers, ts.totalTransfers))
 		}
-		_, _ = fmt.Fprintf(buf, "Elapsed time:  %10ss\n", strings.TrimRight(elapsedTime.Truncate(time.Minute).String(), "0s")+fmt.Sprintf("%.1f", elapsedTimeSecondsOnly.Seconds()))
+		_, _ = fmt.Fprintf(buf, "Elapsed time:  %10s\n", elapsedTime.String())
```
Comment (ncw) [inline fs/accounting/stats.go:388]: This looks like an unrelated change. It will cause the elapsed time to go from `10s` to `10.5s` to `11s` which is visually distracting (and the reason for the original code - though the original code is pretty weird now I look at it!)

**Review by ncw — COMMENTED** [review]
I think this looks like a reasonable idea provided it doesn't make the stats output too long.

By too long I mean > 80 characters ideally.

Can you paste some examples of what the new config looks like?

File: fs/accounting/stats.go (hunk 1 above)
Comment (howeyc) [reply fs/accounting/stats.go:388]: Actually, the original code did that too, I just removed the unnecessary splitting between minutes/seconds that was happening (time.Duration.String() does that just fine). I was actually going to remove tenths-of-second (and only show seconds), but didn't figure it was my place to decide that.

**Comment by howeyc** [comment]
Here's a couple examples:

purge:
0 / 0 Bytes, -, 0 Bytes/s, ETA - (chk#74096/74191, del#74191f,0d) 17m53.1s elapsed
copy:
9.231M / 13.167 MBytes, 70%, 1.007 MBytes/s, ETA 3s (xfr#58/1349) 10s elapsed

**Comment by howeyc** [comment]
Perhaps the word "elapsed" isn't necessary, to keep the line length as short as possible.

**Comment by howeyc** [comment]
A couple examples with shorter units, and second-precision elapsed time.

copy:
269.156M / 39.149 GB, 1%, 4.150 MB/s, ETA 2h39m55s (ck#1349/1349,xr#73/10085) 1m5
purge:
0 / 0 B, -, 0 B/s, ETA - (ck#725450/725545, rm#725545f0d) 3h20m59s

File: fs/accounting/stats.go (hunk 1 above)
Comment (ncw) [reply fs/accounting/stats.go:395]: This is what the original code does

```
$ rclone copy -v --stats 0.5s --bwlimit 1k ~/go/src/github.com/rclone/rclone /tmp/rclone.copy 2>&1 | grep Elapsed
Elapsed time:         0.5s
Elapsed time:         1.0s
Elapsed time:         1.5s
Elapsed time:         2.0s
Elapsed time:         2.5s
Elapsed time:         3.0s
Elapsed time:         3.5s
Elapsed time:         4.0s
```

And this is what your code does

```
$ rclone copy -v --stats 0.5s --bwlimit 1k ~/go/src/github.com/rclone/rclone /tmp/rclone.copy 2>&1 | grep Elapsed
Elapsed time:          0s
Elapsed time:          1s
Elapsed time:          1s
Elapsed time:          2s
Elapsed time:          2s
Elapsed time:          3s
Elapsed time:          3s
Elapsed time:          4s
```

So it seems to have removed the tenths?

**Review by ncw — COMMENTED** [review]
> Here's a couple examples:

I put these up to the 80 char ruler...

```
         1         2         3         4         5         6         7         8         
12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789
-------------------------------------------------------------------------------|
0 / 0 Bytes, -, 0 Bytes/s, ETA - (chk#74096/74191, del#74191f,0d) 17m53.1s elapsed
9.231M / 13.167 MBytes, 70%, 1.007 MBytes/s, ETA 3s (xfr#58/1349) 10s elapsed
269.156M / 39.149 GB, 1%, 4.150 MB/s, ETA 2h39m55s (ck#1349/1349,xr#73/10085) 1m5
0 / 0 B, -, 0 B/s, ETA - (ck#725450/725545, rm#725545f0d) 3h20m59s
```

Pretty good!

Removing the word `elapsed` seems like a good idea.

I personally like the decimal seconds as when you are using -P you get to see it change every output (since the default output time is 0.5s) but I don't think it is a big loss.

So I think I'm happy with the changes.

Are you happy now too? If so I'll merge.

**Comment by howeyc** [comment]
Yes, looks good to me.

**Comment by ivandeex** [comment]
@ncw
After recent Stats patches this PR needs a little conflict resolving.
The submitter just closed it.
Do you think it's reasonable to recover?
